src/components/molecules/ProfilePictureInput/ProfilePictureInput.tsx

```diff
@@ -11,10 +12,11 @@ import {
 import { resizeFile } from "utils/image";
 
 import "./ProfilePictureInput.scss";
+import { useQuery } from "hooks/useQuery";
```

Comment:
Import ordering

```suggestion
import { resizeFile } from "utils/image";

import { useQuery } from "hooks/useQuery";

import "./ProfilePictureInput.scss";
```

Reply:
Changed in: https://github.com/sparkletown/sparkle/commit/6cdbdb2d81f61c67a0665f91bf358d09fc89bfbe